README.md

```diff
@@ -49,7 +59,69 @@ The `fairlearn` package contains the following algorithms for mitigating unfairn
 > _Note_:
 > DP refers to demographic parity, EO to equalized odds, and BGL to bounded group loss. For more information on these and other terms we use in this repository please refer to the [terminology page](./TERMINOLOGY.md). To request additional algorithms or fairness definitions, please open a [new issue](https://github.com/fairlearn/fairlearn/issues).
 
-## Install fairlearn
+
+### Fairlearn dashboard
+
+Fairlearn dashboard is a Jupyter notebook widget for assessing how a model's predictions impact different groups (e.g., different ethnicities), and also for comparing multiple models along different fairness and accuracy metrics.  
+
+#### Set-up and a single-model assessment
+
+To load the visualization dashboard and assess a single model's fairness and accuracy, launch the dashboard widget within a Jupyter notebook as follows:
+
+```python
+from fairlearn.widget import FairlearnDashboard
+
+# A_test containts your protected attributes (e.g., Age, Gender)
+# sensitive_feature_names containts your protected attributes names
+# y_true contains ground truth labels
+# y_pred contains prediction labels
+
+FairlearnDashboard(sensitive_features=A_test,
+                   sensitive_feature_names=['Gender', 'Age'],
+                   y_true=Y_test.tolist(),
+                   y_pred=[y_pred.tolist()])
+```
+
+
+After the launch, the widget walks the user through the assessment set-up, where the user is asked to select (1) the sensitive feature of interest (e.g., gender or age), and (2) the accuracy metric (e.g., model precision) along which to evaluate the overall model performance as well as any disparities across groups. These selections are then used to obtain the visualization of the model's impact on the subgroups (e.g., model precision for females and model precision for males).
+
+The following figures illustrate the setsup steps, where `gender` is selected as a sensitive feature and `accuracy` is selected as the accuracy metric.
+
+
+
+![Dashboard set-up](img/fairlearn-dashboard-config.png)
+
+
+
+After the set-up the dashboard presents the model assessment in two panels:
+
+
```

Comment:
Our figures are only showing _binary gender_, so I thought it was incongruous when the text talks about _female, male, non-binary gender_ categories. I see two fixes:
* either replace the sensitive feature name `Gender` -> `Binary gender` in all the examples (to explicitly acknowledge that the data set is binarizing the gender)
* or include `non-binary` category in our bar charts